### PR TITLE
add new layer Swoole

### PR DESCRIPTION
**Pull request by deminy**
This is to add the PHP extension [Swoole](https://pecl.php.net/package/swoole) as a new layer. The layer has been tested.

### CODE REVIEW

File: layers/swoole/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+ARG PHP_VERSION
+FROM bref/build-php-$PHP_VERSION:1.7.14 AS ext
```
Comment (Nyholm) [inline layers/swoole/Dockerfile:2]: Can you please add

```



ARG PHP_VERSION
ARG BREF_VERSION
FROM bref/build-php-$PHP_VERSION:$BREF_VERSION AS ext
```

See https://github.com/brefphp/extra-php-extensions/blob/1.0.0/layers/redis/Dockerfile

**Comment by Nyholm** [comment]
Thank you. 

Can you please see to update this PR? Lot's of things have happened here in the past 2 weeks

**Comment by deminy** [comment]
When updating my branches, I merged changes without rebasing. Please feel free to let me know if there is any problem with it.

**Comment by Nyholm** [comment]
Thank you

**Pull request merged**